allensdk/brain_observatory/behavior/behavior_project_lims_api.py

```diff
@@ -0,0 +1,303 @@
+import pandas as pd
+from typing import Optional, List, Dict, Any, T
+import logging
+
+from allensdk.brain_observatory.behavior.internal.behavior_project_base\
+    import BehaviorProjectBase
+from allensdk.brain_observatory.behavior.behavior_data_session import (
+    BehaviorDataSession)
+from allensdk.brain_observatory.behavior.behavior_ophys_session import (
+    BehaviorOphysSession)
+from allensdk.internal.api.behavior_data_lims_api import BehaviorDataLimsApi
+from allensdk.internal.api.behavior_ophys_api import BehaviorOphysLimsApi
+from allensdk.internal.api import PostgresQueryMixin
+from allensdk.brain_observatory.ecephys.ecephys_project_api.http_engine import (
+    HttpEngine)
+from allensdk.core.typing import SupportsStr
+
+StrAble = SupportsStr[T]
```

Comment:
Looks like CI fails. Not 100% sure but I think it should be `StrAble = 'SupportsStr[T]'`?

Reply:
Yep I need to do a little more digging. Don't know why this class doesn't exist already.

Reply:
Yep I need to do a little more digging. Don't know why this class doesn't exist already.

Reply:
Looked at the source code for SupportsInt. Fixed the type in the latest push.